editor/src/core/performance/__snapshots__/performance-regression-tests.spec.tsx.snap

```diff
@@ -576,6 +576,29 @@ Array [
   "/Symbol(react.forward_ref)(EmotionCssPropInternal)/div/ChildPinControl/Symbol(react.memo)()",
   "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.memo)(Symbol(react.forward_ref)())",
   "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.memo)(Symbol(react.forward_ref)())",
+  "/Symbol(react.forward_ref)(Styled(div))/div//span",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(Styled(div))",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(Styled(div))",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(UIGridRow)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//span",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(UIGridRow)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//span",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.memo)()",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(UIGridRow)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//span",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.memo)()",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(UIGridRow)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//span",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(UIGridRow)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//span",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.memo)(RadixSelect)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(UIGridRow)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//span",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.memo)(RadixSelect)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(UIGridRow)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//Symbol(react.forward_ref)(Styled(div))",
+  "/Symbol(react.forward_ref)(Styled(div))/div//UtopiaSpiedFunctionComponent(InspectorModal)",
+  "/Symbol(react.forward_ref)(Styled(div))/div//div",
```

Comment:
it looks like we are now rendering a bunch of extra elements even when the advanced modal is closed? that is possibly impacting performance negatively. 

could you add `.displayName` props to the new components, so we can know more? thanks!

Reply:
another leftover from the previous code - fixed 🙏